app/code/Magento/ReviewGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,78 @@
+    reviews(
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once."),
+        currentPage: Int = 1 @doc(description: "Specifies which page of results to return."),
+    ): ProductReviews! @doc(description: "The list of products reviews.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\Reviews")
+}
+
+type ProductReviews {
+    items: [ProductReview]! @doc(description: "An array of product reviews.")
+    page_info: SearchResultPageInfo! @doc(description: "Metadata for pagination rendering.")
+}
+
+type ProductReview @doc(description: "Details of a product review") {
+    product: ProductInterface! @doc(description: "Contains details about the reviewed product") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product")
+    summary: String! @doc(description: "The review summary (a.k.a title")
+    text: String! @doc(description: "The review text.")
+    nickname: String! @doc(description: "The customer's nickname. Defaults to customer name if logged in.")
+    created_at: String! @doc(description: "Date indicating when the review was created.")
+    average_rating: Float! @doc(description: "The average rating for product review.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\Review\\AverageRating")
+    ratings_breakdown: [ProductReviewRating!]! @doc(description: "An array of ratings by rating category. For example quality, price.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\Review\\RatingBreakdown")
+}
+
+type ProductReviewRating {
+    name: String! @doc(description: "The review rating name for example quality, price.")
+    value: String! @doc(description: "The rating value given by customer. Possible values by default: 1 to 5.")
+}
+
+type Query {
+    productReviewRatingsMetadata: ProductReviewRatingsMetadata! @doc(description: "Metadata required by clients to render ratings & reviews section.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\ProductReviewRatingsMetadata")
+}
+
+type ProductReviewRatingsMetadata {
+    items: [ProductReviewRatingMetadata!]! @doc(description: "List of product reviews sorted based on position")
+}
+
+type ProductReviewRatingMetadata {
+    id: String! @doc(description: "Base 64 encoded rating id.")
+    name: String! @doc(description: "The review rating name for example quality, price")
+    values: [ProductReviewRatingValueMetadata!]! @doc(description: "List of product review ratings sorted based on position.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\ProductReviewRatingValueMetadata")
+}
+
+type ProductReviewRatingValueMetadata {
+    value_id: String! @doc(description: "Base 64 encoded rating value id.")
+    value: String! @doc(description: "e.g Good, Perfect, 3, 4, 5")
```

Comment:
These values are the rating's options.
Quality:
- Good
- Perfect

These options are going to be used on ratings' rendering on the storefront.

Reply:
I understand the numerical ratings. But mapping it with Good, Perfect is something I think we should leave it to the clients. Are we gonna provide any mapping terminology for every rating? can you explain it further it will help me understand the context.

Reply:
Looks like I mixed them up. I agree that currently we have only numerical strings, and I was thinking the purpose of Good, Perfect, is something that could be later implemented. 
As far as these are strings, it could be implemented a different type of ratings with different options different than stars.
For example for Quality rating, we can have a dropdown (visually not using stars) with:
- Good
- Perfect
- etc.

What do you think?

Reply:
> For example for Quality rating, we can have a dropdown (visually not using stars) with:

> Good
> Perfect
> etc.
> What do you think?

This sounds a bit like changing the design of the feature, which I don't think we're looking to do at the moment.

It sounds like this part of the schema is meant to model the [configurable product ratings](https://docs.magento.com/user-guide/stores/attributes-rating.html#create-your-own-ratings).

Agree with @keharper that it's not super clear to a client how to use this data right now